lib/blacklight/xml_parser.rb

```diff
@@ -113,7 +113,8 @@ def self.get_master_parent(pre_data, parents_ids, parent_id)
   # non-metadata file.
   ##
   def self.iterate_files(zipfile)
-    files = zipfile.glob("*/**/**")
+    files = zipfile.entries.select(&:file?)
```

Comment:
I agree. If you were to change zipfile.glob to `zipfile.glob("**/**/**")` then it would catch everything including the current directory. But I'm not sure that is what we want. As that includes the imsmanifest.xml and the .dat files. But for sure the glob isn't capturing the directories.